apps/civicsignalblog/eslint.webpack.config.js

```diff
@@ -23,4 +23,8 @@ module.exports = {
     },
     extensions: [".js"],
   },
+  typescript: {
+    alwaysTryTypes: false,
+    project: "./",
```

Comment:
What is the difference when specifying `project: "./"` vs project: `"./tsconfing.json"`?